api/types/types_drivers_executor.go

```diff
@@ -46,6 +46,18 @@ const (
 
 	// LSXCmdMounts is the command for getting a list of mount info objects.
 	LSXCmdMounts = "mounts"
+
+	// LSXCmdVolumeCreate is the command for creating a volume.
+	LSXCmdVolumeCreate = "volumeCreate"
+
+	// LSXCmdVolumeRemove is the command for removing a volume
+	LSXCmdVolumeRemove = "volumeRemove"
+
+	// LSXCmdVolumeAttach is the command for attaching a volume.
+	LSXCmdVolumeAttach = "volumeAttach"
+
+	// LSXCmdVolumeDeatch is the command for detaching a volume.
+	LSXCmdVolumeDeatch = "volumeDetach"
```

Comment:
Hi @codenrhoden,

Sorry, I wasn't trying to be snarky. I had not refreshed the page and did not see your follow-up. The kids do not have school, and I'm busy watching She-Ra with my daughter while my son is at speech therapy. :)